products/rhel10/profiles/ism_o.profile

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+metadata:
+    SMEs:
+        - shaneboulden
+        - wcushen
+        - eliseelk
+        - sashperso
+        - anjuskantha
+
+reference: https://www.cyber.gov.au/ism
+
+title: 'Australian Cyber Security Centre (ACSC) ISM Official'
```

Comment:
please add Base / Secret / Top Secret to profile titles. That will allow users to distinguish between them when titles are listed